modules/cucumber/modules/runner/src/docker/resources/soapui-log4j.xml

```diff
@@ -0,0 +1,105 @@
+
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/" debug="false">
+
+    <!-- =================== -->
+    <!-- Appenders           -->
+    <!-- =================== -->
+
+    <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
+        <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
+        <param name="Target" value="System.out"/>
+        <param name="Threshold" value="INFO"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <!-- The default pattern: Date Priority [Category] Message\n -->
+            <param name="ConversionPattern" value="%d{ABSOLUTE} %-5p [%c{1}] %m%n"/>
+        </layout>
+    </appender>
+
+    <appender name="FILE" class="org.apache.log4j.RollingFileAppender">
+        <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
+        <param name="File" value="${soapui.logroot}soapui.log"/>
+        <param name="Threshold" value="INFO"/>
+        <param name="Append" value="false"/>
+        <param name="MaxFileSize" value="5000KB"/>
+        <param name="MaxBackupIndex" value="50"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%d %-5p [%c{1}] %m%n"/>
+        </layout>
+    </appender>
+
+    <appender name="ERRORFILE" class="org.apache.log4j.RollingFileAppender">
+        <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
+        <param name="File" value="${soapui.logroot}soapui-errors.log"/>
+        <param name="Threshold" value="INFO"/>
+        <param name="Append" value="true"/>
+        <param name="MaxFileSize" value="5000KB"/>
+        <param name="MaxBackupIndex" value="50"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%d %-5p [%c{1}] %m%n"/>
+        </layout>
+    </appender>
+
+    <appender name="SOAPUI" class="com.eviware.soapui.support.log.SoapUIAppender">
+        <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
+    </appender>
+
+    <appender name="GLOBAL_GROOVY_LOG" class="org.apache.log4j.FileAppender">
+        <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler"/>
+        <param name="File" value="${soapui.logroot}global-groovy.log"/>
+        <param name="Threshold" value="DEBUG"/>
+        <param name="Append" value="true"/>
+        <!-- <param name="MaxFileSize" value="5000KB"/>
+            <param name="MaxBackupIndex" value="50"/> -->
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%d %-5p [%c{1}] %m%n"/>
+        </layout>
+    </appender>
+
+    <logger name="groovy.log">
+        <level value="INFO"/>
+        <appender-ref ref="GLOBAL_GROOVY_LOG"/>
+    </logger>
+    <!-- =============== -->
+    <!-- Loggers         -->
+    <!-- =============== -->
+
+    <logger name="com.eviware.soapui">
```

Comment:
Most of the appenders in this log4j config are for local execution. Do we really want to print SoapUI logs when executing recipes? I think we should add appender for cucumber step defs and add logging messages in those classes.